dipy/workflows/base.py

```diff
@@ -60,18 +60,35 @@ def __init__(self, prog=None, usage=None, description=None, epilog=None,
         self.doc = None
 
     def add_workflow(self, workflow):
-        specs = inspect.getargspec(workflow)
-        doc = inspect.getdoc(workflow)
-        self.doc = NumpyDocString(doc)['Parameters']
+        """ Take a workflow object and use introspection to extract the parameters,
```

Comment:
From numpy doc
```
"""This is the form of a docstring.

It can be spread over several lines.

"""
```
There is no need to indent on the second line. More here
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt